### PR TITLE
RFC Add OS/2 magic and dependency parser for LX executables.

### PR DESCRIPTION
**Pull request by ydario**
Added file magic and parser for LX executable format.

### CODE REVIEW

File: fileattrs/os2.attr [hunk 1]
```diff
@@ -0,0 +1,3 @@
+%__os2_provides		%{_rpmconfigdir}/os2deps.sh --provides
+%__os2_requires		%{_rpmconfigdir}/os2deps.sh --requires
+%__os2_magic		^(32|64)-bit.*$
```
Comment (Conan-Kudo) [inline fileattrs/os2.attr]: This magic is too broad. Is there a may to make this a more restrictive match so that it doesn't just get everything?

Comment (ydario) [reply fileattrs/os2.attr]: The full output is 

     32-bit OS/2 executable [WINDOWCOMPAT]

so maybe I can add 'OS/2'

Comment (ydario) [reply fileattrs/os2.attr]: mmhh is it ran across which file types? e.g. DLL  magic is

    32-bit DLL [WINDOWCOMPAT]

Comment (Conan-Kudo) [reply fileattrs/os2.attr]: Would this work? `^(32|64)-bit (OS\/2 executable|DLL).*$`

Comment (ydario) [reply fileattrs/os2.attr]: that would be fine, sure.

Comment (Conan-Kudo) [reply fileattrs/os2.attr]: I haven't tested it, so you should probably test. :)

**Comment by Conan-Kudo** [comment]
The CI returned the following error:

```
Makefile.am:186: error: OS_OS2 does not appear in AM_CONDITIONAL
Makefile.am: installing './depcomp'
fileattrs/Makefile.am:13: error: OS_OS2 does not appear in AM_CONDITIONAL
scripts/Makefile.am:45: error: OS_OS2 does not appear in AM_CONDITIONAL
```

**Comment by Conan-Kudo** [comment]
The error in the CI is due to its dependency on #349.

**Comment by pmatilai** [comment]
@ydario, I didn't promise to merge OS/2 support, I only suggested you address the things that are NOT OS/2 specific.

**Comment by pmatilai** [comment]
Sorry but NAK, we're not going to add OS2 support since it's not something we can even try to promise to keep, see eg #345 (comment) for rationale.

**Pull request closed without merging**